server/src/main/java/com/skillmagnet/Course/Course.java

```diff
@@ -1,47 +1,43 @@
 package com.skillmagnet.Course;
 
-import java.time.LocalDateTime;
-
+import com.skillmagnet.Lesson.Lesson;
+import jakarta.persistence.*;
+import lombok.Getter;
+import lombok.Setter;
 import org.hibernate.annotations.CreationTimestamp;
 
-import jakarta.persistence.Entity;
-import jakarta.persistence.GeneratedValue;
-import jakarta.persistence.Id;
-import jakarta.persistence.Table;
-import lombok.Data;
+import java.time.LocalDateTime;
+import java.util.List;
 
-@Data
+@Getter
+@Setter
 @Entity
 @Table(name = "course")
 public class Course {
+
     @Id
     @GeneratedValue
     private int id;
 
+    @OneToMany(mappedBy = "course", cascade = CascadeType.ALL, fetch = FetchType.LAZY)
+    private List<Lesson> lessons;
```

Comment:
Is it safe to assume we should always use `cascade = CascadeType.ALL`? implementing something similar for enrolls